foreshadow/steps/preparerstep.py

```diff
@@ -263,12 +270,64 @@ class instead of normal Pipelines.  .. #noqa: I102
             **kwargs: kwargs to PIpeline constructor.
 
         """
+        self._parallel_process = None
+        if "_parallel_process" in kwargs:
+            self._parallel_process = kwargs.pop("_parallel_process")
         self.column_sharer = column_sharer
         if self.column_sharer is None:
             self.column_sharer = ColumnSharer()
-        self._parallel_process = None
         super().__init__(**kwargs)
 
+    def configure_column_sharer(self, column_sharer):  # noqa
+        self.column_sharer = column_sharer
+        self._parallel_process.configure_column_sharer(column_sharer)
+
+    def dict_serialize(self, deep=True):  # noqa
```

Comment:
So the transformer_by_column_group is a serialization of the transformer_list of parallel_processor?

Reply:
> So the transformer_by_column_group is a serialization of the transformer_list of parallel_processor?

Yes that's correct.